brainframe_qt/ui/dialogs/license_agreement/license_agreement.py

```diff
@@ -32,8 +32,8 @@ def get_agreement(cls, parent):
         license_md5 = hashlib.md5(dialog.text.encode('utf-8')).hexdigest()
 
         # Check if the license was already agreed to
-        license_accepted = settings.client_license_accepted.val()
-        license_accepted_md5 = settings.client_license_md5.val()
+        license_accepted = LicensingSettings().eula_accepted
```

Comment:
Could you explain the rational here of creating the LicenseSettings object each time you need to get the value?

Reply:
I guess it probably doesn't need to be a new instance, but it doesn't really matter as there are no instance variables to worry about

Reply:
Actually this is a `classmethod` so there's not really an instance to put it on, and it's not called very often